Resources/Prototypes/Entities/Objects/Specific/chemistry-bottles.yml

```diff
@@ -46,6 +46,34 @@
     solution: drink
   - type: StaticPrice
     price: 0
+  - type: DamageOnLand
+    damage:
+      types:
+        Blunt: 5
+  - type: DamageOtherOnHit
+    damage:
+      types:
+        Blunt: 4
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 5
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: GlassBreak
+      - !type:SpillBehavior { }
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          ShardGlass:
+            min: 1
+            max: 1
```

Comment:
Bottles cost 50 glass (half a sheet) to make, or even less with upgrades. Glass shards can be turned into glass sheets by using a welder. So this will let you get infinite glass sheets if you keep printing/breaking/welding.

Reply:
the same can be said for beakers with upgrades

Reply:
well, these are no longer questions for me, as mentioned above - beakers also break into glass, and there are no complaints about them.

Reply:
Beakers cost 100 glass not 50, so it's much less of a problem than this is, since you don't profit any glass unless you upgrade the lathe. And that's getting nerfed soon anyway.

Reply:
anyway either:
- change to min 0 so its 50% glass return
- ignore it since nobodys gonna make a glass farm wtf

Reply:
change it to min 0

Reply:
Done